scripts/test/system.sh

```diff
@@ -68,6 +68,35 @@ test_subctl_gather
 _subctl diagnose all --validation-timeout 20
 _subctl diagnose firewall inter-cluster --validation-timeout 20 "${KUBECONFIGS_DIR}"/kind-config-cluster1 "${KUBECONFIGS_DIR}"/kind-config-cluster2
 
+# Test subctl diagnose in-cluster
+cat <<EOF | kubectl apply -f -
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: submariner-diagnose
+  namespace: submariner-operator
+spec:
+  template:
+    metadata:
+      labels:
+        submariner.io/transient: "true"
+    spec:
+      containers:
+      - name: submariner-diagnose
+        image: quay.io/submariner/subctl:devel
```

Comment:
```suggestion
        image: localhost:5000/subctl:local
```

You'll also need to add `subctl` to `PRELOAD_IMAGES` - https://github.com/submariner-io/subctl/blob/devel/Makefile#L27